CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing
```

Comment:
File format: there is a mix of 80 chars wide wrapping and much longer lines.
To stay consistent, pick the one you prefer (maybe 80 chars so this file is easy to read on github?) and reformat the other lines.

Reply:
done